onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -828,7 +834,7 @@ common::Status TensorrtExecutionProvider::Compile(const std::vector<onnxruntime:
 
       // Run TRT inference
       if (!trt_context->enqueueV2(&buffers[0], nullptr, nullptr)) {
-        return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL, "TensorRT EP Execution Context Enqueue Failed.");
+        return ORT_MAKE_STATUS(ONNXRUNTIME, EP_FAIL, "TensorRT EP Execution Context Enqueue Failed.");
```

Comment:
can you revert this one?
I had left this at FAIL instead of EP_FAIL because maybe it is an intermittent failure. EP_FAIL means the EP will always fail and we should retry using a different EP.